src/touch.js

```diff
@@ -0,0 +1,89 @@
+var common = require('./common');
+var fs = require('fs');
+
+module.exports = _touch;
+
+//@
+//@ ### touch(file)
+//@
+//@ Examples:
+//@
+//@ ```javascript
+//@ touch('file.txt');
+//@ ```
+//@
+//@ Creates a file if it does not exist. Sets the file's mtime and ctime to NOW if the file
+//@ does exist
+function _touch(opts, file) {
+	opts = common.parseOptions(opts, {
+		'a': 'atime',
```

Comment:
Perhaps rename to `atime_only`? That would make more logical sense given the man page:

> ```
> -a     change only the access time
> ```